.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "lwip/lwip"]
 	path = lwip/lwip
-	url = https://github.com/ourairquality/lwip.git
+	url = https://github.com/ccbruce0812/lwip.git
```

Comment:
Not sure about switching across to your own git repo just to get in a modification to upstream code - that is not workable, we all need to work together and try to get changes submitted upstream where possible. You have not even explained why the change is necessary?